pmm-app/src/pmm-qan/panel/components/Filters/Filters.tools.ts

```diff
@@ -1,11 +1,14 @@
 import { getTemplateSrv } from '@grafana/runtime';
-import { FILTERS_GROUPS } from './Filters.constants';
 
-export const getSelectedCheckboxes = (filters) => FILTERS_GROUPS.map((group) => filters[group.dataKey])
-  .filter(Boolean)
-  .map((item) => item.name)
-  .reduce((acc, item) => acc.concat(item), [])
-  .some((item) => item.checked);
+export const getSelectedCheckboxes = (filters, FILTERS_GROUPS) => {
+  const selectedFilters = FILTERS_GROUPS.map((group) => filters[group.dataKey])
```

Comment:
again here - I would not use the misleading naming that indicate that value is a const

Reply:
Fixed with changing the naming.